webserver/main/schemas/schema.json

```diff
@@ -4370,618 +4386,854 @@
-      },
-      "IssueResolution": {
-        "required": [
-          "resolution_action",
-          "resolution_remarks"
-        ],
-        "type": "object",
-        "properties": {
-          "network_issue_id": {
-            "type": "string",
-            "description": "Network issue identifier is a unique number assigned to a complaint by the interfacing application",
-            "format": "uuid"
-          },
-          "resolution": {
-            "type": "string",
-            "description": "resolution"
-          },
-          "resolution_remarks": {
-            "type": "string",
-            "description": "remarks of the resolution of the issue by the respondent"
-          },
-          "gro_remarks": {
-            "type": "string",
-            "description": "resolution remarks provided by the gro"
-          },
-          "dispute_resolution_remarks": {
-            "type": "string",
-            "description": "remarks of the result of the dispute, if applicable"
-          },
-          "resolution_action": {
-            "type": "string",
-            "description": "Indicactes the action taken while addressing the issue.\n ### Actions\n  1. resolve : indicating the issue being resolved. \n  2. reject : indicating that the issue is rejected by the respondant or the resolution provider. \n",
-            "enum": [
-              "RESOLVE",
-              "REJECT"
-            ]
+            "required": [
+              "contact"
+            ],
+            "type": "object",
+            "properties": {
+              "person": {
+                "$ref": "#/components/schemas/Person"
+              },
+              "contact": {
+                "allOf": [
+                  {
+                    "$ref": "#/components/schemas/Contact"
+                  },
+                  {
+                    "required": [
+                      "phone"
+                    ]
+                  }
+                ]
+              }
+            },
+            "description": "- Describes an entity that raises a complaint with an interfacing app. \n- The complainant may be an end user (buyer or seller) or a network participant (buyer app/ seller app/ logistic services  provider app)\n"
           },
-          "action_triggered": {
-            "type": "string",
-            "enum": [
-              "REFUND",
-              "REPLACEMENT",
-              "NO-ACTION"
-            ]
-          }
-        },
-        "description": "Describes the issue resolution for the issue raised by a Complainant"
-      },
-      "ResolutionSupport": {
-        "type": "object",
-        "properties": {
-          "respondentChatLink": {
-            "type": "string",
-            "description": "chatbot link for the respondent for complainant to interact with the respondent"
+          "ResolutionProvider": {
+            "required": [
+              "respondent_info"
+            ],
```

Comment:
same !! are we safe to add required fields

Reply:
@navdeep710 this is as per the IGM updated Yaml. we have not made any changes in the retail schema